mongodb/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+# For local development only, no password required
```

Comment:
No need this file.
We use mongodb atlas on cloud

Reply:
fixed